apps/src/studio/levels.js

```diff
@@ -3077,13 +3077,15 @@ var js_hoc2015_event_free_markdown = [
   '<summary>More about the characters</summary>',
   '<p>Each character has a unique behavior:',
   '<br/><br/>',
+  '<span class="js_hoc2015_event_free_markdown">',
```

Comment:
We had some css that was needed just for this content. This makes that more explicit.